RestComm/NUnitTestClass.cs

```diff
@@ -0,0 +1,16 @@
+﻿using NUnit.Framework;
+using System;
+
+namespace RestComm
+{
+	[TestFixture ()]
```

Comment:
The `TestFixture` gives me the following error when open the project with VisualStudio on Mac
```
/Users/gvagenas/data/devWorkspace/telWorkspace/restcomm-sdk-dotnet/RestComm/NUnitTestClass.cs(3,3): Error CS0619: 'TestFixtureAttribute' is obsolete: 'The NUnit framework shipped with Mono is deprecated and will be removed in a future release. It was based on NUnit 2.4 which is long outdated. Please move to the NUnit NuGet package or some other form of acquiring NUnit.' (CS0619) (RestComm)
```